app/Models/BeatmapsetEvent.php

```diff
@@ -106,6 +107,18 @@ public static function search($rawParams = [])
             }
         }
 
+        if ($searchByBeatmapset) {
+            $params['beatmapset'] = $rawParams['beatmapset'];
+            $beatmapset = Beatmapset::find($params['beatmapset']);
+
+            if ($beatmapset === null) {
+                $query->none();
+            }
+            else {
```

Comment:
```suggestion
            } else {
```
`phpcs` can help you fix minor code style issues like this